packages/react-scripts/template/README.md

```diff
@@ -1327,6 +1294,39 @@ The test command will force Jest to run tests once instead of launching the watc
 
 The build command will check for linter warnings and fail if any are found.
 
+### Pre-commit Hook
+
+You can run tests against "staged" files before each Git commit by integrating the test script in pre-commit hook.
```

Comment:
We already added docs about Prettier (#2006 is merged). I'm referring to consolidating those two sections and removing duplication between them.